### ISSUE TITLE
[UX] Improve "back to folder" button

### ISSUE THREAD

**Issue by ImaCrea** [ONLYOFFICE/DocumentServer#1430]
Hello there, this a UX request about the button "back to folder" :
![Capture d’écran 2021-09-22 à 14 41 18](https://user-images.githubusercontent.com/764332/134346407-786f8019-7d65-4917-852f-7113d8a1cc04.png)

Context : we integrate OnlyOffice into Nextcloud and this button is very useful for our users. Unfortunately, most of the time they don't use it till we tell them it exists. Most of the time people don't even see this button or imagine what it's meant for.

This is simply not where they look for when they want to close the editor.

I was wondering if there's any change coming about that button ?

**Comment by ShockwaveNN** [comment]
You can also use `Open file location` in the File menu

![Screenshot_20210922_155409](https://user-images.githubusercontent.com/668524/134347236-cc9a27af-9329-4a4e-a015-2569fcfc9317.png)

Do you have an alternative placement in mind? Because currently we are not thinking about changing it

**Comment by ImaCrea** [comment]
Thanks for your answer. Yes I can think about a UX suggestion. I'll get back to you soon with something.

**Comment by acsfer** [comment]
Completely agree on this. If that button was replaced by a simple "Save and close" (or other) it would make much more sense.
![image](https://user-images.githubusercontent.com/12234510/134498056-cf66fb5c-a7dc-46c6-8b67-a3bad931573f.png)

Icons only make sense if they are understood **easily**.

**Comment by ShockwaveNN** [comment]
I do not agree with all your points (and this is out of my job responsibilities), but I've created issue 52748 in our private issue tracker to discuss it with our UX team, it's up to them to descide

**Comment by ImaCrea** [comment]
> I do not agree with all your points (and this is out of my job responsibilities), but I've created issue 52748 in our private issue tracker to discuss it with our UX team, it's up to them to descide

Hello there, I was wondering if there might be some news on that side please ? :)

**Comment by ShockwaveNN** [comment]
Sorry. No news

**Comment by ShockwaveNN** [comment]
@ImaCrea Thank you for your work, I'll definitely show your job to our UX team, but I still cannot guarantee any changes because they are completely out of my area of influence

**Comment by ImaCrea** [comment]
> @ImaCrea Thank you for your work, I'll definitely show your job to our UX team, but I still cannot guarantee any changes because they are completely out of my area of influence

With pleasure :)

**Comment by ImaCrea** [comment]
Hello @ShockwaveNN I realize I didn't have any news on that side. 🙃 Could we try a pr with such new icon?

**Comment by ShockwaveNN** [comment]
@ImaCrea Sorry, design team do not make any promises to me, I don't think I can do anything to speed things up

**Comment by pierreozoux** [comment]
It is really crazy to me, to see the simple solution that @ImaCrea found, it is really simple and elegant!

Each time I have to use this functionality, now I'm thinking about the little cross on the top right ;)

Hope the design team will like it!

Cheers :)

**Comment by trofim24** [comment]
We've added [close button in our api](https://api.onlyoffice.com/docs/docs-api/usage-api/config/editor/customization/customization-standard-branding/#close) and implemented this feature in our nextcloud integration in [9.9.0 version](https://github.com/ONLYOFFICE/onlyoffice-nextcloud/blob/master/CHANGELOG.md#990).
I think, I can close this issue.
If you have any new ideas, please create new issue. Thanks.
